cmd/common-main.go

```diff
@@ -50,24 +50,42 @@ func initConfig() {
 	}
 }
 
-func handleCommonCmdArgs(ctx *cli.Context) {
-	// Set configuration directory.
-	{
-		// Get configuration directory from command line argument.
-		configDir := ctx.String("config-dir")
-		if !ctx.IsSet("config-dir") && ctx.GlobalIsSet("config-dir") {
-			configDir = ctx.GlobalString("config-dir")
+func handleGlobalCmdArgs(ctx *cli.Context) error {
```

Comment:
Why do you want to change the function name?

Reply:
I wanted it to be explicit that this function handles global flags. I can change it back to the old name if it makes more sense. Let me know.